src/languages/lib/swift.js

```diff
@@ -0,0 +1,285 @@
+import { concat, either } from '../../lib/regex';
+
+// Keywords that require a leading dot.
+export const dotKeywords = [
+  /Protocol\b/, // contextual
+  /Type\b/, // contextual
+];
+
+// Keywords that may have a leading dot.
+export const optionalDotKeywords = [
+  /init\b/,
+  /self\b/
+];
+
+// Regular keywords and literals.
+export const keywords = [
+  /\bAny\b/,
+  /\bSelf\b/,
+  /\b_\b/, // pattern
+  /\bassociatedtype\b/,
+  /\bas\?\B/, // operator
+  /\bas!\B/, // operator
+  /\bas\b/, // operator
+  /\bbreak\b/,
+  /\bcase\b/,
+  /\bcatch\b/,
+  /\bclass\b/,
+  /\bcontinue\b/,
+  /\bconvenience\b/, // contextual
+  /\bdefault\b/,
+  /\bdefer\b/,
+  /\bdeinit\b/,
+  /\bdidSet\b/, // contextual
+  /\bdo\b/,
+  /\bdynamic\b/, // contextual
+  /\belse\b/,
+  /\benum\b/,
+  /\bextension\b/,
+  /\bfallthrough\b/,
+  /\bfalse\b/, // literal
+  /\bfileprivate\(set\)\B/,
+  /\bfileprivate\b/,
+  /\bfinal\b/, // contextual
+  /\bfor\b/,
+  /\bfunc\b/,
+  /\bget\b/, // contextual
+  /\bguard\b/,
+  /\bif\b/,
+  /\bimport\b/,
+  /\bindirect\b/, // contextual
+  /\binfix\b/, // contextual
+  /\binit\?\B/,
+  /\binit!\B/,
+  /\binout\b/,
+  /\binternal\(set\)\B/,
+  /\binternal\b/,
+  /\bin\b/,
+  /\bis\b/, // operator
+  /\blazy\b/, // contextual
+  /\blet\b/,
+  /\bmutating\b/, // contextual
+  /\bnil\b/, // literal
```

Comment:
How do we make sure this is properly tagged as a "literal"?  Ditto for true, false, else... is a separately list not needed?

Reply:
Fixed!